src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -2514,6 +2513,61 @@ void CmdSketcherConstrainDistance::applyConstraint(std::vector<SelIdPair> &selSe
 
         return;
     }
+    case 8: // {SelEdge, SelEdge}
```

Comment:
This case aims to handle Two Edges selection but the One Edge case is triggered first and the selection is flushed.

Reply:
Yes, this is a current limitation. You can't have `{SelEdge}` and `{SelEdge, SelEdge}` for the same command as the former will prevent the later to happen.
This can be improved in the future though. :)